lib/js/src/protocol/_SdlProtocolBase.js

```diff
@@ -38,14 +38,14 @@ import { _FrameType } from './enums/_FrameType.js';
 import { _MessageFrameAssembler } from './_MessageFrameAssembler.js';
 import { _SdlPacket } from './_SdlPacket.js';
 import { _ControlFrameTags } from './enums/_ControlFrameTags.js';
-import { _BitConverter } from './../util/_BitConverter.js';
+import { _BitConverter } from '../util/_BitConverter.js';
```

Comment:
@ymalovanyi @vladmu Do you need unit tests for this?  For example, sdl java suite has `SdlProtocolTests`

Reply:
@santhanamk we don't need the test for this particular line as this is just a shortenest path identical to previous but aligned with the style of other imports.

Reply:
@vladmu I meant do you need unit tests overall for the changes made to this file?

Reply:
@santhanamk it is hard to define expectations of such tests in the order of the library and this class as we don't have any tests of previous functionality covered here. We appreciate any help in this case.

Reply:
@vladmu Ok no problem.  It was an optional request, as these were not existing before.